src/scripts/main.js

```diff
@@ -1,3 +1,11 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (let i = 0; i < rows.length; i++) {
+  const secondCell = rows[i].cells[1];
+  const clonedCell = secondCell.cloneNode(true);
+
+  rows[i].insertBefore(clonedCell, rows[i].cells[rows[i].cells.length - 1]);
+}
```

Comment:
Instead of using a `for` loop, consider using the `Array.prototype.forEach` method. It's more readable and less error-prone. Also, it's better to avoid direct DOM manipulation in a loop. It can cause reflows and repaints which negatively affect performance. Consider creating a DocumentFragment, perform all manipulations on it, and then append it to the DOM.